docker-compose.yml

```diff
@@ -143,7 +149,7 @@ services:
     restart: always
     depends_on: *backend-deps
     environment:
-      <<: *backend-env
+      <<: [*backend-env, *clickhouse-env]
```

Comment:
Why does this worker need the Clickhouse variables?

Reply:
The event export task is executed by this worker https://github.com/cvat-ai/cvat/blob/develop/cvat/apps/events/views.py#L84